Gemfile.lock

```diff
@@ -1,175 +0,0 @@
-GIT
```

Comment:
slightly different gems between OS types. Makes no sense anymore to put it under version control :-(